src/store/index.js

```diff
@@ -30,6 +30,9 @@ const store = new Vuex.Store({
     pinned(state) {
       const result = state.pinStore[state.database]
       return _.isNil(result) ? [] : result
+    },
+    schemaTables(state){
+      return _.chain(state.tables).groupBy('schema').value()
```

Comment:
Not sure what the right name here would be. Don't love `schemaTables`, but hey - naming things is hard :D